Makefile

```diff
@@ -155,10 +155,16 @@ VM_DEP=$(RPMFILE)
 VM_PACKAGE=--install cockpit-ws --install `pwd`/$(RPMFILE)
 endif
 
+ifeq ($(TEST_SCENARIO),rawhide)
+# needs to run as a separate command, as --run-command is executed before --script
+RAWHIDE=bots/image-customize -v --run-command 'dnf install -y fedora-repos-rawhide && dnf update -y --enablerepo=rawhide --releasever=$$(. /etc/os-release; echo $$(( VERSION_ID + 1)) ) podman conmon crun containernetworking-plugins containers-common' $(TEST_OS)
```

Comment:
I thought that just `dnf -y --releasever=33 update podman_and_friends` would be enough. Why do we need `fedora-repo-rawhide`?

Reply:
Because [StackOverflow](https://stackoverflow.com/questions/24968410/install-single-package-from-rawhide) said so :grin: Indeed it works fine without, and that's much cleaner. I'll update it.